bom/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright 2014-2018 Immutables Authors and Contributors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>immutables</artifactId>
+        <groupId>org.immutables</groupId>
+        <version>2.8.9-SNAPSHOT</version>
+    </parent>
+    <artifactId>bom</artifactId>
+    <name>${project.groupId}.${project.artifactId}</name>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>testing</artifactId>
+                <version>2.8.9-SNAPSHOT</version>
```

Comment:
This file will be more maintainable if `2.8.9-SNAPSHOT` is replaced with `${project.version}`.

Reply:
If you are using the maven release plugin or maven versions plugin maven will automatically set the correct versions.

Reply:
Should I update the pull request and replace the explicit version?

Reply:
Let's see what @elucash says, but personally I would, if for no other reason than that (a) it reduces churn and (b) more clearly states intent (namely that these versions should be one and the same.)

We have a company-internal BOM which uses `${project.version}` and have had no issues with that. (The `maven-release-plugin` and `maven-versions-plugin` behave as expected, as do downstream BOM usages.)

Reply:
As far as I know there are no issues with using `${project.version}` neither with maven nor with gradle. At least not in recent versions.

Reply:
My preference would be to use `${project.version}` here: we use vars sparingly, for repeated versions, and I think here is just the right place to use em. Also I stopped using release plugin, use just deploy, a lot less trouble overall and almost the same amount of manual work: replacing versions in editor vs cli. though with vars it will be less occurrences to replace.

Reply:
Yes replacing it with an editor would be error-prone and tedious. I changed it to `${project.version}`.

Reply:
Thank you! that will do.
p.s. In editor, I use search\replace in files, not that difficult and one-off, but having less occurrences and smaller diff helps.

Reply:
Glad I could help.